roles/etcd/molecule/default/molecule.yml

```diff
@@ -0,0 +1,57 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: vagrant
+  provider:
+    name: virtualbox
+platforms:
+    - name: etcd1
+      box: contiv/centos73
```

Comment:
This needs to be pinned to a specific version.

Reply:
done